client/verta/verta/_repository/commit.py

```diff
@@ -31,14 +32,15 @@ class Commit(object):
         ID of the Commit, or ``None`` if the Commit has not yet been saved.
 
     """
-    def __init__(self, conn, repo, parent_ids=None, id_=None, date=None, branch_name=None):
+    def __init__(self, conn, repo, commit_msg, branch_name=None):
         self._conn = conn
+        self._commit_msg = commit_msg
```

Comment:
It looks like this protobuf object is only being stored to hang onto the Commit's message for `__repr__()`. Can we just store that instead? Otherwise ID and date are duplicated in the protobuf and in `self.id, self.date` and it's unclear to me which attribute should be referred to.

Reply:
Let's remove them from self then. It doesn't make sense to me to have to add code in 3 locations (from_id call to constructor, constructor arguments and setting inside constructor) every time we want to add a new field.